src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_getAssessmentsByCrn.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.justice.digital.delius.data.api.OffenderAssessments;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+public class OffendersResource_getAssessmentsByCrn extends IntegrationTestBase {
+    @Test
+    public void canGetOffenderAssessmentsByCrn() {
+        final var offenderAssessments = given()
+                .auth()
+                .oauth2(tokenWithRoleCommunity())
+                .contentType(APPLICATION_JSON_VALUE)
+                .when()
+                .get("/offenders/crn/X320741/assessments")
+                .then()
+                .statusCode(200)
+                .extract()
+                .body()
+                .as(OffenderAssessments.class);
+
+      assertThat(offenderAssessments.getRsrScore()).isEqualTo(1);
+    }
+
+
+
+    @Test
+    public void canGetOffenderAssessmentsByCrn_offenderNotFound_returnsNotFound() {
+        given()
+                .auth()
+                .oauth2(tokenWithRoleCommunity())
+                .contentType(APPLICATION_JSON_VALUE)
+                .when()
+                .get("/offenders/crn/X777777/assessments")
+                .then()
+                .statusCode(404);
+    }
+
+}
```

Comment:
Possibly add a new integration test, see OffenderIdentifiersAPITest as a model example. You might need to amend the seed data to add an RSR score for offender X320741 or whatever. Advantage of doing a full test is it will prove your JPA mapping all works as well

Reply:
https://github.com/ministryofjustice/community-api/pull/330/files#diff-cc243f1da52d34856b3b298f0ef17ba1230abfc0a48d9c9888f9c7293b4f148b

This is an integration test - I think it covers the JPA mapping. It's in the same package as OffenderIdentifiersAPITest and looks pretty similar

Reply:
Of course